ROMFS/px4fmu_common/init.d/airframes/1002_standard_vtol.hil

```diff
@@ -54,39 +54,36 @@ param set-default VT_TYPE 2
 
 param set-default CA_AIRFRAME 2
 param set-default CA_ROTOR_COUNT 5
-param set-default CA_ROTOR4_AX 1
-param set-default CA_ROTOR4_AZ 0
-param set-default CA_ROTOR4_PX 0.2
-
-# Square quadrotor X PX4 numbering
-param set-default CA_ROTOR_COUNT 4
```

Comment:
just to check.. this is not needed in this airframe conf anymore?

Reply:
these
``
param set-default CA_ROTOR4_AX 1
param set-default CA_ROTOR4_AZ 0
param set-default CA_ROTOR4_PX 0.2
``
are still needed. But `param set-default CA_ROTOR_COUNT 4` is wrong and duplicated of `param set-default CA_ROTOR_COUNT 5` before